update an author info; add a blog post about phonfieldwork

agricolamz opened the pull request.

steffilazerte commented:
Hi @agricolamz! 
I'll be your editor for this article :grin: Could you checkout the [Pre-submission Checks in the Blog Guide](https://blogguide.ropensci.org/technical.html#presubchecks), specifically making sure to add the checklist for authors to this pull request. Once you've got that in place I conduct my review. Thanks!

agricolamz commented:
Hi, @steffilazerte, here it is:

* [x] I have read the Content Guidelines.
* [x] I have read the Technical Guidelines.
* [x] I used or followed the R Markdown or Markdown template.
* [x] I have followed the Style Guide.
* [x] I created or updated my author metadata with correct folder name.
* [x] I have added relevant tags after browsing existing tags.
* [x] I ran `roblog::ro_lint_md()` on index.md (optional).
* [x] I ran `roblog::ro_check_urls()` on index.md (optional).
* [x] I ran a spell-check on index.md.
* [x] I have added the tags - Software Peer Review, my-packagename, R.
* [x] I have added the package-version YAML tag.
* [x] I have added acknowledgement of the reviewers' work.
* [x] I have added a link to the software peer review thread.
* [x] I ran a spell-check on index.md.

steffilazerte commented:
Awesome thanks! I'll review later today or tomorrow morning.

steffilazerte commented on content/blog/2020-12-22-phonfieldwork-phonetic-fieldwork-and-experiments/index.Rmd:
```diff
@@ -0,0 +1,193 @@
+  - sound analysis
+  - fieldwork
+  - phonfieldwork
+  - lingtypology
```
```suggestion
  - lingtypology
  - community
```

steffilazerte reviewed (commented):
Hi @agricolamz!

Thank you for submitting your nicely organized and written blog post. The area of phonetics and linguistics is new to me, but I'm quite familiar with sound analysis of bird songs. I can therefore imagine how useful it is it have an R package that helps with the fieldwork workflow!

The post is clear, well-written and useful, and the examples nicely illustrate your packages utility. I particularly like your introduction. The idea of having all these bits and pieces of scripts that you want to pass on to others, only to find that they no longer work is definitely something I can relate to and I imagine many others can as well!

The only question I had about the content, was that you list the steps in linguistic research and specify that phonfieldwork applies to steps 3, 5, 8 and partially 4. Which examples apply to step 3? Did I miss that or is that not included as an example?

Other than that small point, I have 19 minor comments/suggestions throughout the pull request. Most are stylistic and a couple are wording suggestions to improve clarity.

Unless it's a checklist item (see below), feel free to accept, modify or reject the suggestions. This is your post so you have the final say!

We're doing well for time, so if I could get your comments by end of day Thursday, December 17th, that'd be great.

Thanks again, and let me know if you have any questions! 

P.S. I realize that some of these items were not as specific on the Author's checklist, I've opened an issue in the Blog Guide to correct that. 

* [X] post follows Content Guidelines
* [x] post follows Style Guide
  - **Remove formatting of package names throughout**
  - **Use "https://docs.ropensci.org/phonfieldwork" for linking to docs**
* [X] title is in Title Case
* [X] publication date is ok
* [x] alternative text of images is informative
   - **Alt text should be a description of the image for screen readers (see my suggestions)**
* [X] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [x] I ran `roblog::ro_check_urls()` on index.md
  - **No problems, but I'll run this again after the links are updated, just in case :grin:**
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok
* [X] YAML package-version included
* [x] YAML subject tags - software peer review, packagename, R; add "community" for post by non-staff non-editor
   - **Add community tag**
* [x] acknowledges and links to reviewers
  - **Add links to reviewers**
* [X] links to peer review thread

agricolamz commented:
@steffilazerte,

thank you very much for you comments:

> The only question I had about the content, was that you list the steps in linguistic research and specify that phonfieldwork applies to steps 3, 5, 8 and partially 4. Which examples apply to step 3? Did I miss that or is that not included as an example?

I didn't thought that the blog post will illustrate all steps, but I decided that it is a nice idea. I added two small paragraphs in order to do it.

> Use "https://docs.ropensci.org/phonfieldwork" for linking to docs

I change it in most cases, but you can find some links to the old version of documentation. It is not a mistake, I intentionally keep some of theme not within too beautiful ropensci `pkgdown` pages. Here is [an issue](https://github.com/ropensci/phonfieldwork/issues/32), where we discuss it.

steffilazerte reviewed (commented):
Hi @agricolamz,
Great additions and thanks for the speedy revisions!
I have 5 final comments for you and then we're good to go: One relates to a change that I'm not sure was intentional or finished, another adds a more detailed alt to your new image, a small  wording suggestion, and the final two are my fault: I gave you the wrong suggestion for changing to the author links (they need a leading "/")

Thanks!

agricolamz commented on content/blog/2020-12-22-phonfieldwork-phonetic-fieldwork-and-experiments/index.Rmd:
```diff
@@ -0,0 +1,226 @@
+
+There was a goal to be able to convert multiple sound annotation formats into data.frame format, so you can find a whole bunch of functions that serve this purpose:
+
+* convert Praat `.TextGrid` files[^6]; see also [`rPraat`](https://fu.ff.cuni.cz/praat/#rpraat-package-for-r) and [`textgRid`](https://github.com/patrickreidy/textgRid) packages
```
corrected.

agricolamz commented:
Dear @steffilazerte,

I think, I resolved all problems. Thank you!

steffilazerte commented:
Awesome, thanks @agricolamz, we're good to go! Below is a draft tweet to advertise your post when it is published last week. Let me know if you have any suggestions or comments.

---

[blog] Phonetic Fieldwork and Experiments with the phonfieldwork Package for R

@agricolamz demonstrates phonfieldwork, a pkg supporting "science craft" (field work, data management, etc.) in the field of linguistics.

https://ropensci.org/blog/2020/12/22/phonfieldwork-phonetic-fieldwork-and-experiments/

#rstats #phonfieldwork #linguistics

agricolamz commented:
@steffilazerte, thank you! It looks good. I'd add the #phonetics tag.

steffilazerte commented:
Good idea, I've edited the tweet accordingly, thanks 😄
